build.gradle

```diff
@@ -3,7 +3,7 @@ task wrapper(type: Wrapper) {
 }
 
 allprojects {
-    version = "3.0.2"
+    version = "3.0.3"
```

Comment:
this should be done separately to make it easier to revert PRs

Reply:
yes it should